packages/atomic/src/components/atomic-search-interface/atomic-search-interface.tsx

```diff
@@ -275,6 +283,18 @@ export class AtomicSearchInterface {
     this.engine.executeFirstSearchAfterStandaloneSearchBoxRedirect(analytics);
   }
 
+  private engineIsCreated(engine?: SearchEngine): engine is SearchEngine {
```

Comment:
The `engine` argument is superfluous, the method could just check `if(!this.engine)`

Reply:
I know but the type is better after, dont need `this.engine!`